### PR TITLE
view documentation locally

### PR DESCRIPTION
**Pull request by AndrewChubatiuk**
- changed docs to a new format
- added ability to debug documentation changes locally

### CODE REVIEW

**Comment by f41gh7** [comment]
Looks like it also should resolve https://github.com/VictoriaMetrics/operator/issues/1038

File: docs/FAQ.md [hunk 1]
```diff
@@ -8,15 +8,12 @@ menu:
...
+  - /operator/faq/
+  - /operator/faq/index.html
 ---
-
-# FAQ (Frequency Asked Questions)
-
 ## How do you monitor the operator itself?
 
-You can read about vmoperator monitoring in [this document](./monitoring.md).
+You can read about vmoperator monitoring in [this document](./monitoring/).
```
Comment (Haleygo) [inline docs/FAQ.md]: But those relative paths are not valid on github, no?

Comment (AndrewChubatiuk) [reply docs/FAQ.md]: you can run `make docs-debug` and check it at http://localhost:1313

Comment (AndrewChubatiuk) [reply docs/FAQ.md]: @Haleygo fixed links, now they are working fine for both github and for vmdocs

**Review by Haleygo — APPROVED** [review]
LGTM!

**Review by Haleygo — APPROVED** [review]

**Pull request merged**